src/RingDecomposerLib/CMakeLists.txt

```diff
@@ -28,4 +28,4 @@ build_library("${SHARED_LIBRARY}" "SHARED")
 build_library("${STATIC_LIBRARY}" "STATIC")
 
 install(FILES RingDecomposerLib.h
-        DESTINATION include)
+        DESTINATION RingDecomposerLib)
```

Comment:
my bad...
```suggestion
        DESTINATION include)
```

Reply:
err, yeah, I should have caught that too. That's what I get for blindly trusting you! ;-)